### ISSUE TITLE
alarmTimeagoWarnMins and alarmTimeagoUrgentMins are returning as strings.

### ISSUE THREAD

**Issue by someoneAnyone** [nightscout/cgm-remote-monitor#3623]
Please include the following information with your bug report:

Is this a new bug with the latest release, have you seen this earlier?
Latest release. 0.10.2-release-20171201

Which device / browser version was used to reproduce the bug?
Every browser... this is an API endpoint issue.
/api/v1/status.json

In previous releases, the JSON output of the status endpoint returned `alarmTimeagoWarnMins`  & `alarmTimeagoUrgentMins` as numbers/doubles but now it is returning strings. This is crashing the app Nightscouter and seems like a bug. I could code around it in Swift but it seems kinds of a bad way to fix the issue.

```
{"status":"ok","name":"Nightscout","version":"0.10.2-release-20171201","serverTime":"2018-06-09T00:17:03.640Z","serverTimeEpoch":1528503423640,"apiEnabled":true,"careportalEnabled":false,"boluscalcEnabled":false,"head":"","settings":{"units":"mg/dl","timeFormat":12,"nightMode":false,"editMode":true,"showRawbg":"always","customTitle":"Staging Nightscout","theme":"colors","alarmUrgentHigh":true,"alarmUrgentHighMins":[30,60,90,120],"alarmHigh":true,"alarmHighMins":[30,60,90,120],"alarmLow":true,"alarmLowMins":[15,30,45,60],"alarmUrgentLow":true,"alarmUrgentLowMins":[15,30,45],"alarmUrgentMins":[30,60,90,120],"alarmWarnMins":[30,60,90,120],"alarmTimeagoWarn":true,"alarmTimeagoWarnMins":**"15"**,"alarmTimeagoUrgent":true,"alarmTimeagoUrgentMins":**"30",**"language":"en","scaleY":"log","showPlugins":" delta direction upbat rawbg","showForecast":"ar2","focusHours":3,"heartbeat":60,"baseURL":"","authDefaultRoles":"readable","thresholds":{"bgHigh":120,"bgTargetTop":109,"bgTargetBottom":90,"bgLow":60},"DEFAULT_FEATURES":["bgnow","delta","direction","timeago","devicestatus","upbat","errorcodes","profile"],"alarmTypes":["simple","predict"],"enable":["rawbg","delta","direction","upbat","maker","treatmentnotify","bgnow","timeago","devicestatus","errorcodes","profile","simplealarms","ar2"]},"extendedSettings":{},"authorized":null}
```

**Comment by someoneAnyone** [comment]
Is anyone supporting this project?

**Comment by PieterGit** [comment]
@someoneAnyone Yes, we are supporting this project. But we are all volunteers.  Did you try this with the `dev` branch?  You mean this used to work with Habanero Yellow (0.10.1) release?

Could you try to create a PR for Nightscout dev to get this issue fixed? If you need help please let use know, or join https://gitter.im/nightscout/public

**Comment by someoneAnyone** [comment]
Volunteer, here too. Trying to keep [Nightscouter](https://github.com/someoneAnyone/Nightscouter) working. FYI - I'm looking for [help](https://github.com/someoneAnyone/Nightscouter/issues) too! ;-)

I'm seeing the issue in 0.10.2-release-20171201 but not 0.10.0-release-20170922
I just built  0.10.3-dev-20180624 and thankfully I think it is resolved.

**Comment by someoneAnyone** [comment]
This is how I ended up solving it in the client app. https://github.com/someoneAnyone/Nightscouter/commit/c79365f3a0f4854868261df542a5b03df5d59a10

**Comment by PieterGit** [comment]
So it's a regression from 0.10.0-release-20170922 to  0.10.2-release-20171201. The `dev` branch will soon be merged as a new master and a 0.10.3 release.
@someoneAnyone can you confirm that nothing needs to be done in the `dev` branch, for this issue? In that case I close it because it will be fixed in the 0.10.3 release.

**Comment by PieterGit** [comment]
@someoneAnyone Closing this issue, as this issue seems resolved in the 0.10.3 release. Please reopen if I'm not right.
